catalogue/webapp/pages/works.js

```diff
@@ -452,9 +391,18 @@ Works.getInitialProps = async (ctx: Context): Promise<Props> => {
       })
     : null;
 
+  const workTypeAggregations = shouldGetWorks
```

Comment:
Could we make this async please as we need to make sure it's performant before releasing, I will make sure it's prioritised.